src/test/harness.rs

```diff
@@ -23,6 +23,7 @@ use languageserver_types as ls_types;
 use serde_json;
 use crate::server as ls_server;
 use rls_vfs::Vfs;
+use lazy_static::lazy_static;
```

Comment:
Am I correct that we don't have any particular preferred import order in RLS?

I personally really love when imports follow `std:: / extern:: / crate::` order. If I am not the only one with obsession for ordering things, I can do the cleanup job sometime in the future :)

Reply:
I don't mind too much, personally I tend to use a single clump rustfmt-ed as it's easy to maintain.

In any case we should rustfmt the project agan, although this can cause annoying merge disruption to prs.

Reply:
@matklad sounds good! I assume Rustfmt does not support splitting imports into these 3 sections?

Reply:
rustfmt leaves whatever sections of imports you use